improver_tests/acceptance/test_generate_realizations.py

```diff
@@ -55,7 +54,6 @@ def test_percentiles(tmp_path):
     acc.compare(output_path, kgo_path)
 
 
-@pytest.mark.slow
 def test_percentiles_reordering(tmp_path):
```

Comment:
There is one instance of `@pytest.mark.slow` in this file that can now be removed, as all tests take less than 1 second.